File: LandscapeController_ObjC/LandscapeController_ObjC/SecondViewController.m
//
//  SecondViewController.m
//  Simple_Player_ObjC
//
//  Created by olxios on 26/08/14.
//  Copyright (c) 2014 swiftiostutorials.com. All rights reserved.
//

#import "SecondViewController.h"

@interface SecondViewController ()

@end

@implementation SecondViewController
@synthesize isPresented;

- (void)viewDidLoad
{
    [super viewDidLoad];
    self.isPresented = YES;
}

- (IBAction)dismiss:(id)sender
{
    isPresented = NO;
    [self.presentingViewController dismissViewControllerAnimated:YES completion:nil];
}

@end

File: LandscapeController_Swift/LandscapeController_Swift/SecondViewController.swift
//
//  SecondViewController.swift
//  LandscapeController_Swift
//
//  Created by olxios on 20/06/16.
//  Copyright © 2016 olxios. All rights reserved.
//

import UIKit

class SecondViewController: UIViewController {
    
    var isPresented = true
    
    @IBAction
    func dismiss() {
        
        isPresented = false
        self.presentingViewController!.dismissViewControllerAnimated(true, completion: nil);
    }

}
